checks.d/http_check.py

```diff
@@ -83,7 +84,7 @@ def _check(self, instance):
                 auth = (username, password)
 
             r = requests.get(addr, auth=auth, timeout=timeout, headers=headers,
-                             verify=not disable_ssl_validation)
+                             verify=False if disable_ssl_validation else instance_ca_certs)
```

Comment:
I agree that's a bit confusing but that's how the `requests` library works. From http://docs.python-requests.org/en/latest/user/advanced/#ssl-cert-verification:

>By default, verify is set to True. Requests can ignore verifying the SSL certificate if you set verify to False.
>You can pass verify the path to a CA_BUNDLE file with certificates of trusted CAs. This list of trusted CAs can also be specified through the REQUESTS_CA_BUNDLE environment variable.

`instance_ca_certs` should only be a single string though (not a list), ca_certs is plural because a single file usually contains multiple ca certificates. Do you think we should change the config parameter and variable names to make less confusing?